packages/cdk8s-plus/src/deployment.ts

```diff
@@ -195,21 +176,11 @@ export class DeploymentSpecDefinition {
   public get labelSelector(): Record<string, string> {
     return { ...this._labelSelector };
   }
-
+  
   /**
    * @internal
    */
-  public _toKube(deployment: Deployment): k8s.DeploymentSpec {
-
-    // automatically select pods in this deployment
```

Comment:
So how are the pods in the deployment selected now?

Reply:
You are right. Let me revise.

Reply:
Fixed.